const InternalException: Error = {
  message: 'Internal Exception',
  name: 'InternalException',
} as const

export default InternalException
